webui/src/app/app.component.ts

```diff
@@ -14,10 +14,11 @@ import { LoaderBean, LoadersStoreService } from './stores/loader-store.service';
 import { SidebarModule } from 'primeng/sidebar';
 import { ContentBean } from './models/commons/content-bean';
 import { ErrorBean, ErrorsStoreService, DropErrorAction, NewErrorAction } from './stores/errors-store.service';
-import { BehaviorSubject } from 'rxjs/BehaviorSubject';
+import { BehaviorSubject } from 'rxjs';
 import { ConfigStoreService, ConfigBean } from './stores/config-store.service';
 
 import { each, find } from 'lodash';
+import { Observable } from 'rxjs/internal/Observable';
```

Comment:
Sounds very weird to rely on someone else' internals! Auto-import typo for `rxjs/Observable`?